index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal profile</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+  </head>
+  <body>
+    <div id="video-background">
+      <video autoplay muted loop id="video">
+        <source src="favvideo.mp4" type="video/mp4">
+      </video>
+    </div>
+    <header id="header">
+      <div id="navbar">
+        <div class="icon-container">
+          <a id="profile-link" href="index.html" target="_blank"><i class="fa fa-fw fa-home"></i></a>
+          <span class="icon-text">Home</span>
+        </div>
+        <div class="icon-container">
+          <a href="about.html"><i class="fa fa-fw fa-user"></i></a>
+          <span class="icon-text">About</span>
+        </div>
+        <div class="icon-container">
+          <a href="contact.html"><i class="fa fa-fw fa-envelope"></i></a>
+          <span class="icon-text">Contact</span>
+        </div>
+        <div class="icon-container">
+          <a href="#welcome-section-text"><i class="fa solid fa-file"></i></a>
+          <span class="icon-text">Projects</span>
+        </div>
+      </div>
+    </header>
```

Comment:
- Remove the Target blank from your page links as it is internal page links you use target blank when you want to link some external websites.